src/js/profile.js

```diff
@@ -170,6 +170,19 @@ $(function() {
         window.location = window.location.origin + "/collection.html#" + id
     }
 
+    function deleteColl(evt){
+      var id = this.parentNode.parentNode.parentNode.dataset.collectionId
```

Comment:
This is ok. A better way is to identify a class name or something that can be used to find the relevant node, but that might have to rely on jquery.